src/diem/testing/cli/click.py

```diff
@@ -51,9 +61,25 @@ def start_server(
 )
 @click.option("--test-debug-api", "-d", default=False, help="Run tests for debug APIs.", type=bool)
 @click.option("--verbose", "-v", default=False, help="Enable verbose log output.", type=bool, is_flag=True)
-def test(target: str, jsonrpc: str, faucet: str, pytest_args: str, test_debug_api: bool, verbose: bool) -> None:
+def test(
+    target: str,
+    host: str,
+    port: Optional[int],
+    base_url: Optional[str],
+    jsonrpc: str,
+    faucet: str,
+    pytest_args: str,
+    test_debug_api: bool,
+    verbose: bool,
+) -> None:
     configure_testnet(jsonrpc, faucet)
     os.environ[envs.TARGET_URL] = target
+    os.environ[envs.DMW_STUB_BIND_HOST] = host
+    if port:
+        os.environ[envs.DMW_STUB_BIND_PORT] = str(port)
+    if base_url:
+        os.environ[envs.DMW_ACCOUNT_BASE_URL] = base_url
```

Comment:
DMW_STUB_DIEM_ACCOUNT_BASE_URL

Reply:
for the case we are running 2 miniwallet, one is target, another one is stub, the name "DMW_ACCOUNT_BASE_URL" maybe confusing, as we don't know which wallet is using it.